sidecar/nginx/conf/nginx.conf

```diff
@@ -26,12 +26,12 @@ http {
   keepalive_requests 10000;
   types_hash_max_size 2048;
 
-  include /etc/nginx/conf.d/*.http.conf;
+  include /etc/nginx/conf.d/*.conf;
```

Comment:
ugh.. Good point. this is the default nginx config.. But we also need to be able to let people pull in stream config stuff. May be we put the stream files under conf.d/stream/*.conf ?